lib/lotus/view/rendering/partial_finder.rb

```diff
@@ -42,6 +45,14 @@ def find
         end
 
         protected
+        # @since 0.6.0
+        # @api private
+        def find_cached_template
+          partials = Lotus::View.configuration.partials[[view_template_dir, template_name].join(separator)] ||
```

Comment:
Thanks for all the helpful comments @jodosha 

This is the part that I found most difficult to understand. I've changed this to use `Lotus::View::Configuration.for(@view)`. Not sure if it is quite right yet.

CC @pascalbetz I think this is the reason that the sample application you created would have not benefitted from the cache.